src/config/routes.ts

```diff
@@ -1,34 +1,33 @@
 export default {
-  index: () => '/',
-  searchOverlay: () => `/search`,
-  search: ({ query }: { query?: string } = {}) => {
-    const basePath = '/search'
+  viewer: {
+    index: () => '',
```

Comment:
```suggestion
    index: () => '/',
```
I think it should fix the issue mentioned by @mikkio-j